src/main/webapp/user-page.html

```diff
@@ -32,16 +32,6 @@
     </nav>
     <h1 id="page-title">User Page</h1>
 
-    <div id="message-container">Loading...</div>
-    <form id="message-form" action="/messages" method="POST" class="hidden">
-      Enter a new message:
-      <br/>
-      <textarea name="text" placeholder="Enter a message" id="message-input"></textarea>
-      <br/>
-      <input type="submit" value="Submit">
-    </form>
-    <hr/>
-  
     <b>About Me:</b>
     <div id="about-me-container">Loading...</div>
     <br/>
```

Comment:
May possibly want to also get rid of this about-me-container, for consistency? (Unless there is a reason for it to still be, in which case that is more important)

Reply:
Also wondering if we still need this file, if we now have a user-page.jsp (legitimately not sure)

Reply:
Just merged! And yes, thank you for pointing that out! I had forgotten to remove the file. I had a couple of strange merge conflicts, but I think it should be resolved now. Thank you for checking over my PR!